map/bookmark_manager.cpp

```diff
@@ -2238,7 +2238,11 @@ std::vector<kml::MarkGroupId> BookmarkManager::GetSortedBmGroupIdList() const
 
   std::sort(vec.begin(), vec.end(), [](PairT const & lhs, PairT const & rhs)
   {
-    return lhs.second->GetLastModifiedTime() > rhs.second->GetLastModifiedTime();
+    bool const lVisible {lhs.second->IsVisible()};
+    bool const rVisible {rhs.second->IsVisible()};
```

Comment:
C'mon, ```bool const lVisible = lhs.second->IsVisible();```
It is more natural and readable. I don't agree with pushing {} everywhere ..

Reply:
Why is it needed if we don't modify m_lastModified for visibility on/off?